CMake/HPHPFunctions.cmake

```diff
@@ -104,9 +104,9 @@ function(append_systemlib TARGET SOURCE SECTNAME)
     # for each library append the following line to embed.rc
     # $sectionname RCDATA "$source"
     add_custom_command(TARGET generate_rc
-      COMMAND echo "${SECTNAME} RCDATA \"${SOURCE}\"" >> embed.rc
+      COMMAND echo "\"${SECTNAME}\" RCDATA \"${SOURCE}\"" >> embed.rc
       COMMENT "Adding ${SOURCE} as ${SECTNAME} to embed.rc"
-      VERBATIM)
+      )
```

Comment:
The changes above break cygwin build and result in error like "invalid embed format". This may need an if-else block for cygwin and msvc.